cdc/server.go

```diff
@@ -56,11 +53,7 @@ const (
 
 // Server is the capture server
 type Server struct {
-	captureV2 *capture.Capture
-
-	capture      *Capture
-	owner        *Owner
-	ownerLock    sync.RWMutex
+	captureV2    *capture.Capture
```

Comment:
Could you rename 'captureV2' to 'capture'?

Reply:
done